exploration-module/src/main/java/io/edpn/backend/exploration/application/domain/Message.java

```diff
@@ -1,5 +1,17 @@
 package io.edpn.backend.exploration.application.domain;
 
-public record Message(String topic,
-                      String message) {
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder
+public class Message {
+    private String topic;
+    private String message;
```

Comment:
Records not usable for some reason? :disappointed:

Reply:
reverted to records